<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Yandex_lyceum</class>
 <widget class="QMainWindow" name="Yandex_lyceum">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>-2</y>
      <width>421</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="registration_label">
       <property name="text">
        <string>–†–µ–≥–∏—Å—Ç—Ä–∞—Ü–∏—è</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>–§–∞–º–∏–ª–∏—è</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="surname"/>
       </item>
       <item>
        <widget class="QLabel" name="error_surname">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>–í–≤–µ–¥–∏—Ç–µ —Ñ–∞–º–∏–ª–∏—é!</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>–ò–º—è</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="name"/>
         </item>
         <item>
          <widget class="QLabel" name="error_name">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>–í–≤–µ–¥–∏—Ç–µ –∏–º—è!</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>–õ–æ–≥–∏–Ω</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLineEdit" name="login"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="error">
         <property name="text">
          <string>–í–≤–µ–¥–∏—Ç–µ –ª–æ–≥–∏–Ω!</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>–ü–∞—Ä–æ–ª—å</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="password"/>
             </item>
             <item>
              <widget class="QToolButton" name="style_password_button">
               <property name="text">
                <string>üëÅ</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>–ü–∞—Ä–æ–ª—å –µ—â–µ —Ä–∞–∑</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="pwd_check"/>
             </item>
             <item>
              <widget class="QToolButton" name="style_pwd_check_button">
               <property name="text">
                <string>üëÅ</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="error_check1">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>–ü–∞—Ä–æ–ª–∏ –Ω–µ —Å–æ–≤–ø–∞–¥–∞—é—Ç</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="error_check2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>–ü–∞—Ä–æ–ª—å –¥–æ–ª–∂–µ–Ω —Å–æ–¥–µ—Ä–∂–∞—Ç—å –±—É–∫–≤—ã, —Ü–∏—Ñ—Ä—ã, —Å–ø–µ—Ü—Å–∏–º–≤–æ–ª—ã, –±—ã—Ç—å –¥–ª–∏–Ω–µ–µ 8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="error_login">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>–¢–∞–∫–æ–π –ª–æ–≥–∏–Ω —É–∂–µ –∑–∞–Ω—è—Ç</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>–ù–∞–∑–∞–¥</string>
    </property>
   </widget>
   <widget class="QPushButton" name="registration_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>135</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>–ó–∞—Ä–µ–≥–∏—Å—Ç—Ä–∏—Ä–æ–≤–∞—Ç—å—Å—è</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
